src/components/calcite-stepper-item/calcite-stepper-item.tsx

```diff
@@ -244,6 +244,7 @@ export class CalciteStepperItem {
 
   private getItemContent(): HTMLElement[] | NodeListOf<any> {
     // handle ie and edge
+    // todo: Not sure how to remove this IE/Edge code
```

Comment:
Maybe we can just leave this for now? I know stepper needs to be reworked to use a MutationObserver instead of its current logic.